apps/src/lib/kits/maker/Thermometer.js

```diff
@@ -0,0 +1,34 @@
+/** @file wrap Playground.Thermometer to expose raw value */
+import Playground from 'playground-io';
+
+// All we care about doing here is caching the temp sensor's raw input value
+// and then exposing it as a property on the Thermometer object.
+// Otherwise we're going to pass everything through to the playground-io controller.
+
+let thermometerRawValue;
+const Thermometer = {
+  initialize: {
+    value: function () {
```

Comment:
I think I could `Playground.Thermometer.initialize.value.call(this, ...args)`?  The binding is important here, and TBH I'm nervous about deviating from how the PlaygroundIO controller does things because the whole setup seems fragile.